ddsketch/pb/sketchpb/ddsketch.pb.go

```diff
@@ -3,26 +3,26 @@
 // Copyright 2021 Datadog, Inc.
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.27.1
```

Comment:
Thanks for fixing this. If we upgrade `google.golang.org/protobuf` to `v1.28.0`, can we regenerate this with `protoc-gen-go v1.28.0` please?

Reply:
Done. No changes other than the version number.